vmdb/app/controllers/ops_controller/diagnostics.rb

```diff
@@ -133,6 +133,8 @@ def log_depot_edit
         replace_right_cell(x_node)
       end
     when "validate"
+      id = params[:id] ? params[:id] : "new"
```

Comment:
@brandondunne we only set id and load edit from session[:edit] when validate/save are pressed not for cancel/reset buttons